storage/post-storage-logs-to-log-analytics/PostStorageLogs2LogAnalytics.ps1

```diff
@@ -246,70 +227,69 @@ Function ConvertLogLineToJson([String] $logLine)
 }
 
 $storageAccount = Get-AzStorageAccount -ResourceGroupName $ResourceGroup -Name $StorageAccountName -ErrorAction SilentlyContinue
-if($storageAccount -eq $null)
+if($null -eq $storageAccount)
```

Comment:
Can you elaborate on the change to line 230? The logic seems reversed and the change seems unnecessary.

Reply:
Sure, I just follow the PSSscriptAnalyser rules. You are right, this one is not necessary. I got used to fixing such comparisons automatically.
https://github.com/PowerShell/PSScriptAnalyzer/blob/b1a81874fbe61ebaf1a0b9631f337b4f06296625/RuleDocumentation/PossibleIncorrectComparisonWithNull.md